packages/core/src/render3/i18n/i18n_parse.ts

```diff
@@ -69,8 +69,8 @@ export function i18nStartFirstCreatePass(
     tView: TView, parentTNodeIndex: number, lView: LView, index: number, message: string,
     subTemplateIndex: number) {
   const rootTNode = getCurrentParentTNode();
-  const createOpCodes: I18nCreateOpCodes = [];
-  const updateOpCodes: I18nUpdateOpCodes = [];
+  const createOpCodes: I18nCreateOpCodes = [] as any;
```

Comment:
Can we avoid `as any` here (and in other similar cases)?
```suggestion
  const createOpCodes = [] as I18nCreateOpCodes;
```

Reply:
unfortunately no. you can't cast `[]` to `I18nCreateOpCodes` directly without going through `any`.